web/src/app/pages/create-survey/loi-permissions/loi-permissions.component.ts

```diff
@@ -35,8 +43,12 @@ interface OptionCardConfig {
   templateUrl: './loi-permissions.component.html',
   styleUrls: ['./loi-permissions.component.scss'],
 })
-export class LoiPermissionsComponent {
-  readonly loiPermissionsControlKey = 'loiPermissionsMethod';
+export class LoiPermissionsComponent implements OnInit, OnChanges {
+  @Input() permissionsOption!: LoiPermissionsOption;
+  @Output() permissionsOptionSelected: EventEmitter<LoiPermissionsOption> =
```

Comment:
Following naming convention in Angular docs, I think this would be called `select`. Note I'm intentionally omitting "loiPermission" since it's clear from context. ,and since its name is likely to change once we clarify behavior of this feature (#1188).